dist/javascript-calculator.js

```diff
@@ -195,6 +195,7 @@ var global=typeof self !== "undefined" ? self : typeof window !== "undefined" ?
       var answerText, expText;
       this.typeLastPressed = "equals";
       expText = this.element.find("figure.jc--display").html();
+      this.updateCurrentExpressionWithCommand(this.math.commands.build_exit_sub_expression());
```

Comment:
This is the logic applied after right paranthesis click 
Refer https://github.com/thinkthroughmath/javascript-calculator/pull/32/files#diff-5868cae851e5b624f700fc4ef68a43cf0474d7cf5d3b0205ac3074e9de0228aeR227

Line 227 in this file

Reply:
👍